### PR TITLE
Updated project for bootstrap 4, alpha 2

### PR DESCRIPTION
**Pull request by justin808**
* Added support for postcss and tether for Bootstrap 4.
* Added font-awesome example usage (no glyphicons in Bootstrap 4).

Things to do someday:
* It's pretty non-dry for the examples.
* I couldn't get the extract-text-plugin to work with font-awesome unless I removed the url-loader limit of 10000.

[Review on Reviewable](https://reviewable.io/reviews/shakacode/bootstrap-loader/56)

### CODE REVIEW

**Comment by justin808** [comment]
@lazzrek @alexfedoseev @RazkoSA please review.

**Comment by lazzrek** [comment]
Thanks for creating this Pull request @justin808 . I found some issues:
- Duplicate 'postcss-loader' in both package.json files.
- Is it not redundant to require "postcss" if its already installed via postcss-loaders dependency?
- Move all loaders to "devDependencies"

**Comment by justin808** [comment]
@lazzrek Can you please attach your comments to lines of code. Click the "Files Changed" tab or go into Reviewable. Thanks.

File: examples/basic/webpack.prod.config.js [hunk 1]
```diff
@@ -20,13 +26,26 @@ module.exports = {
...
     loaders: [
       { test: /\.css$/, loader: ExtractTextPlugin.extract('style', 'css!postcss') },
       { test: /\.scss$/, loader: ExtractTextPlugin.extract('style', 'css!postcss!sass') },
-      { test: /\.(woff2?|ttf|eot|svg)$/, loader: 'url?limit=10000' },
+
+      {
+        test: /\.woff2?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
+        // Limiting the size of the woff fonts breaks font-awesome ONLY for the extract text plugin
+        // loader: "url?limit=10000"
+        loader: "url"
```
Comment (justin808) [inline examples/basic/webpack.prod.config.js]: @alexfedoseev @wmadden We need to figure this one out. We should be able to have the url limit when doing a prod (extract-text-plugin) deployment.

* http://stackoverflow.com/questions/34133808/webpack-ots-parsing-error-loading-fonts
* http://stackoverflow.com/a/34133809/1009332
* http://stackoverflow.com/a/30763977/1009332

**Comment by alexfedoseev** [comment]
Looks good, merging :+1:

**Pull request merged**